src/lib.rs

```diff
@@ -42,3 +43,6 @@ pub use ioctls::{KvmRunWrapper, Result};
 /// It can be used for calls to [get_supported_cpuid](struct.Kvm.html#method.get_supported_cpuid) and
 /// [get_emulated_cpuid](struct.Kvm.html#method.get_emulated_cpuid).
 pub const MAX_KVM_CPUID_ENTRIES: usize = 80;
+
+/// Maximum number of MSR entries that can be returned by a call to KVM ioctls.
```

Comment:
Please add similar details as the `MAX_KVM_CPUID_ENTRIES` have. Where did you get the value from? What can it be used for?